repository/managers.py

```diff
@@ -30,6 +30,62 @@ def groups(self):
         else:
             response.raise_for_status()
 
+    def group_items(self, groupId):
```

Comment:
do I understand this correctly, that this method returns **all** texts in a group? and then the view paginates? we have groups with multiple thousand texts in it, so this is not the right approach. Vogon should only get the page it wants to show from Citesphere.